addons/interact_menu/functions/fnc_keyDown.sqf

```diff
@@ -37,27 +37,30 @@ GVAR(useCursorMenu) = (vehicle ACE_player != ACE_player) ||
                       {(_menuType == 1) && {(isWeaponDeployed ACE_player) || GVAR(AlwaysUseCursorSelfInteraction) || {cameraView == "GUNNER"}}} ||
                       {(_menuType == 0) && GVAR(AlwaysUseCursorInteraction)};
 
+// Delete existing controls in case there's any left
+GVAR(iconCount) = 0;
+for "_i" from 0 to (count GVAR(iconCtrls))-1 do {
+    ctrlDelete (GVAR(iconCtrls) select _i);
+    GVAR(ParsedTextCached) set [_i, ""];
+};
+GVAR(iconCtrls) resize GVAR(iconCount);
+
 if (GVAR(useCursorMenu)) then {
-    createDialog QGVAR(cursorMenu);
+    (findDisplay 46) createDisplay QGVAR(cursorMenu); //"RscCinemaBorder";//
+    (finddisplay 91919) displayAddEventHandler ["KeyUp", {[_this,'keyup'] call CBA_events_fnc_keyHandler}];
+    (finddisplay 91919) displayAddEventHandler ["KeyDown", {[_this,'keydown'] call CBA_events_fnc_keyHandler}];
```

Comment:
I'm installing CBA key EH on the menu display. Although a bit hacky, it seems to work fine.